docs/specification/auth/credential.md

```diff
@@ -86,13 +86,15 @@ Clients **MUST** provide credentials through headers during initialization:
   "params": {
     "capabilities": {
       "auth": {
-        "credential": true
+        "credentials": true
       }
     },
-    "auth": {
-      "credentials": {
-        "API-KEY": "api_key",
-        "MISC-PASSWORD": "password"
+    "_meta": {
```

Comment:
Sorry, I meant using the `_meta` field _outside_ of initialization only. 😅

Rationale:
1. It's an essential part of initialization semantics
2. It's an out of band addition to every other request type

I think just `params.auth` is fine during init.

Reply:
Hm, I think I would prefer consistency here. Providing it as part of `_meta` in initialization would be the same as making a request with headers. If the server finds valid credentials, it can present a larger suite of capabilities?

Reply:
The thing is that the initialize request is basically _all_ headers (by analogy).